app/src/components/BackLink.tsx

```diff
@@ -1,14 +1,14 @@
 import { ReactElement } from 'react'
 
-import ButtonLink from '@components/ButtonLink'
+import StyledLink from '@components/StyledLink'
 
 export interface BackLinkProps {
   href: string
 }
 
 export const BackLink = (props: BackLinkProps): ReactElement => {
   const { href } = props
-  return <ButtonLink labelKey="backlinkText" href={href} style="unstyled" />
+  return <StyledLink textKey="backlinkText" href={href} external={false} />
```

Comment:
@karinamzalez I ended up switching the back link back to a styled link instead of using a `<button>` component because I noticed this line in the [USWDS Button component](https://designsystem.digital.gov/components/button/) page:

> Linking between a site’s pages. Use regular links instead.